### PR TITLE
Update tidb and pd configuration template

### PR DESCRIPTION
**Pull request by superlzs0476**
@liubo0127  @LinuxGit  PTAL

### CODE REVIEW

**Comment by superlzs0476** [comment]
@WangXiangUSTC wang  PTAL

**Comment by superlzs0476** [comment]
@disksing PTAL

File: roles/pd/vars/default.yml [hunk 1]
```diff
@@ -53,8 +53,14 @@ log:
...
 
 metric:
+  # prometheus client push interval, set "0s" to disable prometheus.
+  interval: "15s"
+  # prometheus pushgateway address, leaves it empty will disable prometheus.
+  address: ""
 
 schedule:
+  max-merge-region-size: 20
+  max-merge-region-keys: 200000
```
Comment (disksing) [inline roles/pd/vars/default.yml]: I think we were intentional to turn merge off in ansible

Comment (superlzs0476) [reply roles/pd/vars/default.yml]: Merge off now or later? :smile:

Comment (disksing) [reply roles/pd/vars/default.yml]: Region merge is turned off without these 2 lines.

Comment (superlzs0476) [reply roles/pd/vars/default.yml]: @disksing  PTAL

**Comment by superlzs0476** [comment]
Not adjusted.

**Pull request closed without merging**